src/components/fields/image-upload/image-review/image-review.styles.ts

```diff
@@ -63,17 +63,18 @@ export const ReviewCloseButton = styled(IconButton)`
 	outline-style: none;
 	> svg {
 		font-size: 2rem;
-		color: ${V2_Color.Primary};
+		color: ${Colour["bg-primary"]};
 	}
 
-	${V2_MediaQuery.MaxWidth.mobileL}, (orientation: landscape) and (max-height: ${V2_MediaWidths.mobileL}px) {
+	${MediaQuery.MaxWidth.sm}, (orientation: landscape) and (max-height: ${({ theme }) =>
+		Breakpoint["md-min"]({ theme })}px) {
 		height: 2.25rem;
 	}
 `;
 
-export const ReviewTitle = styled(V2_Text.H5)`
-	font-weight: 600;
-	color: ${V2_Color.Primary};
+export const ReviewTitle = styled(Typography.BodyMD)`
+	${Font.Spec["weight-semibold"]};
```

Comment:
for font spec you'll need to specify the css property
```suggestion
	font-weight: ${Font.Spec["weight-semibold"]};
```